content/source/docs/cloud/run/index.html.md

```diff
@@ -76,3 +76,9 @@ In the list of workspaces on Terraform Cloud's main page, each workspace shows t
 For full details about the stages of a run, see [Run States and Stages][].
 
 [Run States and Stages]: ./states.html
+
+## Import
+
+Terraform Cloud does not support remote execution for `terraform import`. For this command the workspace acts only as a remote backend for Terraform state, with all execution occuring on your own workstations or continuous integration workers.
+
+Terraform variables defined in the workspace are available, however environment variables defined in the workspace are not available to `terraform import`, and must be defined within your local execution scope.
```

Comment:
```suggestion
Since `terraform import` runs locally, environment variables defined in the workspace are not available. Any environment variables required by the provider you're importing from must be defined within your local execution scope.
```

Since the `import` command can never really do anything with Terraform variables anyway, this paragraph becomes clearer if we don't mention them.